cvat-ui/src/components/projects-page/actions-menu-items.tsx

```diff
@@ -58,6 +58,12 @@ export default function ProjectActionsItems(
         label: <CVATMenuEditLabel>Assignee</CVATMenuEditLabel>,
     }, 30]);
 
+    menuItems.push([{
+        key: 'edit_organization',
+        onClick: () => startEditField('organization'),
+        label: <CVATMenuEditLabel>Organization</CVATMenuEditLabel>,
+    }, 31]);
```

Comment:
We usually try to reserve weight numbers in the `*1–*9` range for plugins, so it’s better to use `40` here.

Reply:
It makes sense, but changing this weight has 2 disadvantages, IMHO:
1. It requires updating all next weights + in private plugins too, am I right?
1. Is there any reason to insert some action between edit actions? I thought that all edit-related actions should be grouped.